.github/workflows/rules-poetry.yml

```diff
@@ -0,0 +1,78 @@
+name: Rules 
+
+on:
+
+  workflow_call:
+    inputs:
+        os:
+          required: true 
+          type: string 
+        python-version:
+          required: true 
+          type: string
+        poetry-extras:
+          required: false 
+          type: string 
+          default: ""
+        doctests:
+          required: false 
+          type: boolean
+          default: false 
+    secrets:
+      CODECOV_TOKEN:
+        required: true
+
+jobs:
+  build:
+    runs-on: ${{ inputs.os }}
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ inputs.python-version }}
+    - name: Install and configure poetry
+      uses: snok/install-poetry@v1
+      with:
+        virtualenvs-create: true
```

Comment:
@andrea-pasquale was this required for any specific reason?

Otherwise, I'd propose to make this explicitly `false`, and drop the `source $VENV` below.

Reply:
I think that if you drop it the workflows will fail (probably for Windows). If not than we can remove it or come up with a different solution @AleCandido.

Reply:
Ok, I put it to `false` in another workflow, but it is true that I only tried to run it on `ubuntu-latest`. I will have a look to the past runs to see where it is failing and why